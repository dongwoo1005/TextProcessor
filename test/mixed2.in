stdin dropfirst 1 count 1 count e count b doublewords allcaps
There will be
no
single one character
bbbb
