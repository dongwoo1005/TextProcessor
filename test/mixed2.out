H1R2
H1R2
ILL
ILL
3
3
O
O
INGL4
INGL4
N5
N5
HARACT6R
HARACT6R
123
123
